hedera-node/docs/system-accounts-operations.md

```diff
@@ -0,0 +1,187 @@
+| 0x168, 0x169 (0.0.360, 0.0.361)    | ✅ - if address is a payable contract<br/>✅ - if address is an account<br/>✅ - if address is empty, creates account<br/>❌ - if address is a non-payable contract | ❌ - INVALID_FEE_SUBMITTED                                                                                                                                                                                                                                               | ❌ - INVALID_RECEIVING_NODE_ACCOUNT                                                                                                                                                                                                                              |
+| 0x16a → 0x2EE (0.0.362 → 0.0.750)  | ✅ - if address is a payable contract<br/>✅ - if address is an account<br/>✅ - if address is empty, creates account<br/>❌ - if address is a non-payable contract | ❌ - INVALID_FEE_SUBMITTED                                                                                                                                                                                                                                               | ❌ - INVALID_RECEIVING_NODE_ACCOUNT                                                                                                                                                                                                                              |
+| 0x2EF → 0x3E8 (0.0.751 → 0.0.1000) | ✅ - if address is a payable contract<br/>✅ - if address is an account<br/>✅ - if address is empty, creates account<br/>❌ - if address is a non-payable contract | ✅ - if account exists and has receiverSigRequired = false<br/>✅ - if (account exists) and (account has receiverSigRequired = true) and (account is sender)<br/>❗- don’t auto create if it doesn’t exist                                                             | ✅ - if account exists and has receiverSigRequired = false<br/>✅ - if (account exists) and (account has receiverSigRequired = true) and (account is sender)<br/>❗- don’t auto create if it doesn’t exist                                                         |
+| 0x3E8 → ♾️ | ✅ - if address is a payable contract<br/>✅ - if address is an account<br/>✅ - if address is empty, creates account<br/>❌ - if address is a non-payable contract | ✅ - if address is a payable contract<br/>✅ - if address is an account with receiversigrequired = false<br/>✅ - if address is empty, creates account<br/>❌ - if address is a non-payable contract<br/>❌ - if address is an account with receiversigrequired=true | ✅ - if address is a payable contract<br/>✅ - if address is an account with receiversigrequired = false<br/>✅ - if address is empty, creates account<br/>✅ - if address is a non-payable contract<br/>❌ - if address is an account with receiversigrequired=true |
```

Comment:
Typos in receiversigrequired -> should be receiverSigRequired

Reply:
Fixed